app/services/invitation_manager.rb

```diff
@@ -28,7 +29,7 @@ def success?
     def process_emails
       emails.reduce([]) do |bad_emails, email|
         invitation = @user.invitations.new(email: email, status: 0, role: @role)
-        invitation.save ? invitation.send! : bad_emails << email
+        invitation.save ? invitation.send!(@url) : bad_emails << email
```

Comment:
My only idea would be to instantiate the invitation with the url something like:
`invitation = @user.invitations.new(email: email, status: 0, role: @role, url: @url)`